install-binary.sh

```diff
@@ -1,115 +1,34 @@
 #!/usr/bin/env bash
 
 # borrowed from https://github.com/technosophos/helm-template
-
-PROJECT_NAME="helm-unittest"
```

Comment:
why drop this and replace it with the value down below? coulda just left it.

Reply:
Because I asked her to delete as much as possible...